x/concentrated-liquidity/incentives_test.go

```diff
@@ -2798,14 +2659,12 @@ func (s *KeeperTestSuite) TestCollectIncentives() {
 			}
 
 			// System under test
-
-			s.Ctx = s.Ctx.WithBlockTime(tc.positionParams.collectTime)
-			actualIncentivesClaimed, err := clKeeper.CollectIncentives(ctx, tc.positionParams.owner, tc.positionParams.positionId)
+			actualIncentivesClaimed, err := clKeeper.CollectIncentives(ctx, ownerWithValidPosition, DefaultPositionId)
```

Comment:
I did not review test coverage of new logic in-detail but would like to confirm if you think that the new logic is sufficiently covered by the existing tests?

Reply:
IMO it is sufficiently covered because we are going from broader to more specific. For instance, before we had to test that multiple positions get claimed together when they cover the same range and same pool. Now we rely on the unique ID which simplifies the logic.

With that being said, I think the tests are sufficient for now, but we may extend them after our first internal audit.